flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/adaptive/AdaptiveSkewedJoinITCase.scala

```diff
@@ -87,6 +87,13 @@ class AdaptiveSkewedJoinITCase extends AdaptiveJoinITCase {
     checkResult(sql)
   }
 
+  @Test
+  def testJoinWithHashOutput(): Unit = {
+    val sql =
+      "SELECT * FROM (SELECT * FROM T1, T2 WHERE T1.a1 = T2.a2) as T4 LEFT JOIN T3 ON T3.b3 = T4.b1"
+    checkResult(sql)
```

Comment:
Since this verification aims to check the result, how can we know whether this change will take effect for AdaptiveSkewedJoin when the configuration is set to AUTO？

Reply:
> Since we are only checking whether the output result is correct, how can we know if the update has taken effect when the configuration is set to AUTO？

There is no good way, I have manually verified it locally.